src/expressions/plyql.pegjs

```diff
@@ -546,6 +558,7 @@ SubstrToken        = "SUBSTR"i "ING"i? !IdentifierPart { return 'substr'; }
 ExtractToken       = "EXTRACT"i        !IdentifierPart { return 'extract'; }
 ConcatToken        = "CONCAT"i         !IdentifierPart { return 'concat'; }
 LookupToken        = "LOOKUP"i         !IdentifierPart { return 'lookup'; }
+IfNullToken        = "IFNULL"i         !IdentifierPart { return 'fallback'; }
```

Comment:
Overall you seem to have taken to pegjs parsers like a duck to water. I am very impressed.